lib/pierogi.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'order'
+require 'twilio-ruby'
+
+class Pierogi
+
+  attr_reader :menu
+  attr_reader :orders
+
+  def initialize
+    @menu = { "small pierogi" => 1, "large pierogi" => 3, "supersize pierogi" => 4,
```

Comment:
Separate file for menu items? - See Open Closed Principle in https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md